packages/components/src/toolbar-button/README.md

```diff
@@ -0,0 +1,26 @@
+# ToolbarButton
+
+A ToolbarButton can be used to add actions to your block control, usually inside a ToolbarGroup. It has similar features to the [Button](https://developer.wordpress.org/block-editor/components/button/) component. Using `ToolbarButton` will ensure the correct styling for a button in a toolbar, and also that keyboard interactions in a toolbar are consistent with the [WAI ARIA toolbar pattern](https://www.w3.org/TR/wai-aria-practices/#toolbar).
```

Comment:
For linking to components, you want to not include the full domain, since the docs are browsed in multiple contexts. See [the Using Links section](https://developer.wordpress.org/block-editor/contributors/document/#using-links) of the contributing docs.

```suggestion
A ToolbarButton can be used to add actions to your block control, usually inside a ToolbarGroup. It has similar features to the [Button](/packages/components/src/button/README.md) component. Using `ToolbarButton` will ensure the correct styling for a button in a toolbar, and also that keyboard interactions in a toolbar are consistent with the [WAI ARIA toolbar pattern](https://www.w3.org/TR/wai-aria-practices/#toolbar).
```

Reply:
Thanks @mkaz, I haven't read that page, will check it out. I'm committing all this suggestions!